xinference/core/gradio.py

```diff
@@ -29,7 +29,7 @@
 MODEL_TO_FAMILIES = dict(
     (model_family.model_name, model_family)
     for model_family in MODEL_FAMILIES
-    if model_family.model_name != "baichuan"
+    if model_family.model_name not in ["baichuan", "llama-2"]
```

Comment:
This judgment also needs to be done in the client: 
https://github.com/xorbitsai/inference/blob/81a085cc76c69a73ff703b48c66cb649675e09f3/xinference/client.py#L317
https://github.com/xorbitsai/inference/blob/81a085cc76c69a73ff703b48c66cb649675e09f3/xinference/client.py#L411